benchmark/solkz/compositional_field/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ ENDIF ()
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 SET(TARGET "solkz_compositional_field")
+SET(TARGET_DEPENDENCY "solkz")
+
 PROJECT(${TARGET})
 
+INCLUDE_DIRECTORIES("..")
+
 ADD_LIBRARY(${TARGET} SHARED solkz_compositional_field.cc)
-ASPECT_SETUP_PLUGIN(${TARGET})	
+ADD_LIBRARY(${TARGET_DEPENDENCY} SHARED ../solkz.cc)
```

Comment:
Hopefully this isn;t cryptic but after changing the line to `MESSAGE("This is ${Aspect_DIR}")` in CMakeLists.txt; here is the output:

```
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$ cmake -DAspect_DIR=/home/hlokavar/build/aspect/master .
CMake Error: An attempt was made to access a variable: ASPECT_DIR that has not been defined. This variable is created by the FIND_PACKAGE command. CMake version 1.6 always converted the variable name to upper-case, but this behavior is no longer the case.  To fix this you might need to set the cache value of CMAKE_BACKWARDS_COMPATIBILITY to 1.6 or less.  If you are writing a CMake listfile, you should change the variable reference to use the case of the argument to FIND_PACKAGE.
This is 
-- Setting up plugin:
--   name <solkz_compositional_field>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Setting up plugin:
--   name <solkz>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Configuring incomplete, errors occurred!
See also "/home/hlokavar/runs/solkz_direct/test/compositional_field/CMakeFiles/CMakeOutput.log".
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$ 
```

It looks like there is a conflict with the argument 
`-DAspect_DIR=/home/hlokavar/build/aspect/master`

As an aside, has there been a similar case where the Aspect source directory is required at compile time?